crates/burn-train/src/metric/hamming.rs

```diff
@@ -0,0 +1,153 @@
+use super::state::{FormatOptions, NumericMetricState};
+use super::{MetricEntry, MetricMetadata};
+use crate::metric::{Metric, Numeric};
+use burn_core::tensor::{activation::sigmoid, backend::Backend, ElementConversion, Int, Tensor};
+
+/// The hamming score, sometimes referred to as multi-label or label-based accuracy.
+pub struct HammingScore<B: Backend> {
+    state: NumericMetricState,
+    threshold: f32,
+    sigmoid: bool,
+    _b: B,
```

Comment:
PhantomData should be used to wrap B. It's zero size type. See https://doc.rust-lang.org/std/marker/struct.PhantomData.html

Reply:
Copied that from the accuracy metric. You're right, will fix both.